source/app.d

```diff
@@ -168,9 +202,28 @@ void main()
 		auto udbsettings = new UserManSettings;
 		udbsettings.serviceName = "DUB - The D package registry";
 		udbsettings.serviceURL = URL("http://code.dlang.org/");
-		udbsettings.serviceEmail = "noreply@vibed.org";
+		udbsettings.serviceEmail = "noreply@rejectedsoftware.com";
```

Comment:
wouldn't this be better configurable or `@code.dlang.org`?

Reply:
Configurable definitely makes sense. But `@code.dlang.org` doesn't have a mail server configured, so I just resorted to something that was already there.